options/options_test.go

```diff
@@ -247,6 +247,7 @@ func TestLoading(t *testing.T) {
 				RevocationProviderRefreshInterval: defaultRevokeProviderRefreshInterval,
 				HashingSalt:                       defaultHashingSalt,
 				RevocationRefreshTolerance:        defaultRevocationRereshTolerance,
+				JwtProcessors:                     make(map[string]interface{}),
```

Comment:
As offline discussed, try to refactor that into a defined interface, that your specific type is implementing.